### PR TITLE
Create and Set Up Add level dialog for Activities Editor GUI

### PR DESCRIPTION
**Pull request by dmcavoy**
This splits out some of the work that was in https://github.com/code-dot-org/code-dot-org/pull/36627. You can watch the gif below to see what happens when you click add level. The dialog that shows when you click add level is all the components in this PR.

activitiesEditor

## Links

- [spec](https://docs.google.com/document/d/1LVwA7VtxkkIFZSi7StwAtgHfxFUIAN1E3nbDRjEXWP4/edit?ts=5f57c117#)
- [jira](https://codedotorg.atlassian.net/browse/PLAT-279)

## Testing story

Added simple unit tests for each component.

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/lib/levelbuilder/lesson-editor/CreateNewLevelInputs.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import React, {Component} from 'react';
+import Button, {ButtonColor} from '@cdo/apps/templates/Button';
+
+//TODO: Set up creating a new level. Make sure dropdown has all level types
+
+export default class AddLevelFilters extends Component {
```
Comment (davidsbailey) [inline apps/src/lib/levelbuilder/lesson-editor/CreateNewLevelInputs.jsx:6]: please fix class name

File: apps/src/lib/levelbuilder/lesson-editor/AddLevelDialog.jsx [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+              </button>
+            </ToggleGroup>
+            {this.state.methodOfAddingLevel === 'Find Level' && (
+              <div style={styles.filtersAndLevels}>
+                <AddLevelFilters />
+                <AddLevelTable addLevel={this.props.addLevel} />
+              </div>
+            )}
+            {this.state.methodOfAddingLevel === 'Create New Level' && (
+              <CreateNewLevelInputs />
```
Comment (davidsbailey) [inline apps/src/lib/levelbuilder/lesson-editor/AddLevelDialog.jsx:100]: can you please include a screenshot of what this looks like?

**Review by davidsbailey — APPROVED** [review]
Nice work, Dani! Thank you for separating this out.

File: apps/src/lib/levelbuilder/lesson-editor/CreateNewLevelInputs.jsx (hunk 1 above)
Comment (uponthesun) [reply apps/src/lib/levelbuilder/lesson-editor/CreateNewLevelInputs.jsx:6]: Huh, how does the code even work as-is? Is it because of `export default`?

**Review by uponthesun — APPROVED** [review]

File: apps/src/lib/levelbuilder/lesson-editor/CreateNewLevelInputs.jsx (hunk 1 above)
Comment (davidsbailey) [reply apps/src/lib/levelbuilder/lesson-editor/CreateNewLevelInputs.jsx:6]: exactly. importing the default implicitly sets the name of the import.

File: apps/src/lib/levelbuilder/lesson-editor/AddLevelDialog.jsx (hunk 2 above)
Comment (dmcavoy) [reply apps/src/lib/levelbuilder/lesson-editor/AddLevelDialog.jsx:100]: Screen Shot 2020-08-28 at 2 39 40 PM

**Pull request merged**